app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -7,9 +7,16 @@
 
 .govuk-govspeak {
   .example {
-    border-left: 10px solid $panel-colour;
-    padding-left: 1.5em;
-    margin: 2.5em 0;
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
```

Comment:
I'm 👍  for using `em` in general but is this consistent with our spacing as defined here: https://govuk-elements.herokuapp.com/layout/?

> spacing between elements is usually 5px, 10px, 15px or multiples of 15px


I'd prefer we redefine our spacing system to use `em` rather than be inconsistent.